DEVELOPING.md

```diff
@@ -83,6 +83,62 @@ if *debugging {
 }
 ```
 
+## Git Workflow
+1.  Fork in the cloud
+    1.  Visit https://github.com/google/go-flow-levee
+    2.  Click `Fork` button (top right) to establish a cloud-based fork.
+
+2.  Clone fork to local storage
+
+    ```bash
+    cd $(go env GOPATH)/src/
+    git clone https://github.com/<your-github-username>/go-flow-levee.git
+
+    cd go-flow-levee
+    git remote add upstream https://github.com/google/go-flow-levee.git
+
+    # Never push to upstream master
+    git remote set-url --push upstream no_push
+
+    # Confirm that your remotes make sense:
+    git remote -v
+    ```
+
+3.  Branch
+
+    Get your local master upto date:
+
+    ```bash
+    cd $(go env GOPATH)/src/go-flow-levee
+    git fetch upstream
+    git checkout master
+    git merge upstream/master
```

Comment:
Should we use rebase instead?

Reply:
And so begins the great debate of rebase vs merge. 😂🤣😂🤣

Reply:
Since we squash when merging to the main branch, there is no difference when it comes to repo history.

I don't use rebase, but I assume it may cause similar issues to force pushing, e.g. cause GitHub to lose track of commits/comment threads?